Assets/BossRoom/Scripts/Game/Client/UI/IPConnectionWindow.cs

```diff
@@ -47,29 +46,17 @@ void Hide()
 
         public void ShowConnectingWindow()
         {
-            var chosenTransport = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().IpHostTransport;
-            NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
-
             void OnTimeElapsed()
             {
                 m_ConnectStatusPublisher.Publish(ConnectStatus.StartClientFailed);
                 Hide();
                 m_IPUIMediator.DisableSignInSpinner();
             }
 
-            switch (chosenTransport)
-            {
-                case UNetTransport unetTransport:
-                    StartCoroutine(WaitUntilUNETDisconnected(OnTimeElapsed));
-                    break;
-                case UnityTransport unityTransport:
-                    var maxConnectAttempts= unityTransport.MaxConnectAttempts;
-                    var connectTimeoutMS= unityTransport.ConnectTimeoutMS;
-                    StartCoroutine(DisplayUTPConnectionDuration(maxConnectAttempts, connectTimeoutMS, OnTimeElapsed));
-                    break;
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(chosenTransport));
-            }
+            var utp = (UnityTransport) NetworkManager.Singleton.NetworkConfig.NetworkTransport;
```

Comment:
I'm pretty sure the format pass will complain about that space after the cast. Same with another instance below.